.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "printWidth": 120,
-  "bracketSpacing": true
+  "bracketSpacing": true,
+  "explicitTypes": "preserve"
 }
 
```

Comment:
Some other useful parameters we could also add if you wanted are:
```
useTabs": false,
"singleQuote": false,
"bracketSpacing": false,
"tabWidth": 4
```
To enforce that all tabs are consistent number of spaces and that all code is formatted in the same way.

Reply:
As discussed IRL:
- `useTabs`, `singleQuote` match the prettier defaults.
- We already have bracket spacing enabled, and we slightly prefer that in js.
- I added explicit tab widths (even though they match the defaults) to the config for clarity.